benchmarks/benchmark_baseline.txt

```diff
@@ -1,9 +0,0 @@
-fill_dense:
- * flat_range                x86_64    83.230 ms       cuda        1.284 ms
- * flat_struct               x86_64     4.662 ms       cuda        2.192 ms
- * nested_range              x86_64    11.367 ms       cuda        1.222 ms
- * nested_range_blocked      x86_64     4.242 ms       cuda        6.570 ms
- * nested_struct             x86_64    12.860 ms       cuda       17.888 ms
-mpm2d:
- * range                     x86_64     0.764 ms       cuda        0.045 ms
- * struct                    x86_64     0.747 ms       cuda        0.206 ms
```

Comment:
Is this file unnecessary?

Reply:
> Is this file unnecessary?

Hi @xumingkuan and others, given that we are having this course GAMES201 real soon and the number of users is highly likely to go up significantly, I think it's a good idea to have a bunch of reference benchmarks for different os/backends/hardware combination. It could help the new users to pick it up more quickly. Do we have a plan like that? I'd love to hear all your opinions. 😄

Reply:
Yes, I guess it only works on @yuanming-hu's machine.